server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -252,7 +252,7 @@
 
   private final Deque<ByteBuffer> walByteBufferPool = new LinkedList<>();
 
-  private int currentWalPoolSize = 0;
+  private AtomicInteger currentWalPoolSize = new AtomicInteger(0);
```

Comment:
Why change the type from `int` to `AtomicInteger`? `synchronized` key word has already ensure the concurrent safety

Reply:
I didn't see the synchronized key word. Already fixed.